lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -630,7 +630,7 @@ sub decline_warning {
 sub show_summary_and_accept_changes {
     my ($self) = @_;
     $self->get_expert_partitioner_page()->press_next_button();
-    $self->get_summary_page()->press_next_button();
+    $self->get_summary_page()->press_next();
```

Comment:
we also need the replacement [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13291/commits/7d709120384b0ee966e70ad67c0414ddfaba73c8#diff-0ffd80c95fb51756da5f9d8a80e01b4f6c2a14898cf18aa594456bc79cccffcdR404) and in the line above, I think.

Reply:
Could you please tell me what is "here" referring to? The link point to the beginning of my second commit.

Reply:
Nevermind, I understood what you meant

Reply:
in line 632 you still have `press_next_button();` and the 'here' that seems I pasted wrongly was in `$self->get_add_volume_group_page()->press_next_button();` in the same file.